TestForSignUp/InputTextGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TestForSignUp
+{
+    internal class InputGenerator
+    {
+        internal string emailString;
+        internal string dateString;
+        internal string commonString;
+
+        internal InputGenerator()
+        {
+            emailString = Guid.NewGuid().ToString() + "@gmail.com";
+            commonString = "A";
```

Comment:
What is this line needed for ?

Reply:
Do you mean 14?

Reply:
Yes, If it is name and last name it doesn't pass validation

Reply:
No, because we added letters to this string on line 15